CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # nf-core/smrnaseq: Changelog
 
+## v1.0.1dev - 2020-03-31
```

Comment:
No date needed for `dev` versions - only when we release.

Reply:
Also, I guess you need to run `nf-core bump-versions` to update to `v1.0.1dev` everywhere..